### PR TITLE
[TM-38] - Enter additional details task

### PR DESCRIPTION
**Pull request by DannyGB**
Requires config from : https://github.com/SkillsFundingAgency/das-employer-config/tree/TM-38-Enter-application-details
and Apim endpoint changes from : https://github.com/SkillsFundingAgency/das-apim-endpoints/tree/TM-38-Enter-application-details
Also requires either the das-courses-api to be running locally or config pointed at one of the test env's

### CODE REVIEW

File: src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/OpportunitiesOrchestrator.cs [hunk 1]
```diff
@@ -186,5 +200,45 @@ private async Task<CreateApplicationCacheItem> RetrievePledgeCacheItem(Guid key)
...
+                Year = application.StartDate?.Year,
+                HasTrainingProvider = application.HasTrainingProvider,
+                OpportunitySummaryViewModel = new OpportunitySummaryViewModel
+                {
+                    Description = GenerateDescription(applicationDetails.Opportunity, request.EncodedPledgeId),
+                    Amount = applicationDetails.Opportunity.Amount,
+                    JobRoleList = string.Join(", ", applicationDetails.Opportunity.JobRoles),
+                    LevelList = string.Join(", ", applicationDetails.Opportunity.Levels),
+                    SectorList = string.Join(", ", applicationDetails.Opportunity.Sectors),
+                    YearDescription = "2021/22"
```
Comment (ConorTill) [inline src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/OpportunitiesOrchestrator.cs:226]: Is this YearDescription fine to be hardcoded? Seen this pop up in a few places.

Comment (DannyGB) [reply src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/OpportunitiesOrchestrator.cs:226]: Yes, I believe we are OK to hard code it. Maybe we should hard code it in only once place though

**Review by chrisfoster76 — CHANGES_REQUESTED** [review]
Some general observations from running the app locally; this is not a code review

File: src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/ApplicationDetails.cshtml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+                <span class="govuk-caption-m">Search and apply for pledges in England</span>
+                <h1 class="govuk-heading-xl">Apprenticeship training</h1>
+
+            </div>
+        </div>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                
+                <partial name="StandardSearch" for="SelectStandardViewModel" />
```
[the remark this answers is not in the document]
Comment (DannyGB) [reply src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/ApplicationDetails.cshtml:21]: This has been fixed following Conors feedback from an earlier review

**Review by chrisfoster76 — APPROVED** [review]

**Pull request merged**